custom_components/tahoma/__init__.py

```diff
@@ -132,10 +134,24 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         "update_listener": entry.add_update_listener(update_listener),
     }
 
+    def has_state(states, names):
+        return any(state.qualified_name in names for state in states)
+
     for device in tahoma_coordinator.data.values():
         platform = TAHOMA_TYPES.get(device.widget) or TAHOMA_TYPES.get(device.ui_class)
         if platform:
             entities[platform].append(device)
+            if device.definition.states and has_state(
+                device.definition.states,
+                [
+                    "core:BatteryState",
+                    "core:BatteryLevelState",
+                    "core:SensorDefectState",
+                ],
+            ):
+                new_device = deepcopy(device)
+                new_device.label = f"BATTERY_{new_device.label}"
```

Comment:
Could you perhaps add a screenshot to this PR? I am not sure if we should add `BATTERY` in front of it?

We should have a look at other integrations, like Z-Wave. They just add `battery` after the label. (not sure how this works with translations)

![image](https://user-images.githubusercontent.com/1424596/104048783-203a0a00-51e4-11eb-83af-4d0c096b8279.png)

Reply:
renamed and move "battery" to the end.